client/client.go

```diff
@@ -483,106 +508,152 @@ type ClientCreateStoreOptions struct {
+	data, _, err := client.OpenFgaApi.CreateStore(request.GetContext()).Body(openfga.CreateStoreRequest{
+		Name: request.GetBody().Name,
 	}).Execute()
 	if err != nil {
 		return nil, err
 	}
 	return &data, nil
 }
 
-func (client *OpenFgaClient) CreateStore(ctx _context.Context) SdkClientCreateStoreRequest {
-	return SdkClientCreateStoreRequest{
-		Client: *client,
+func (client *OpenFgaClient) CreateStore(ctx _context.Context) SdkClientCreateStoreRequestInterface {
+	return &SdkClientCreateStoreRequest{
+		Client: client,
 		ctx:    ctx,
 	}
 }
 
 // / GetStore
 type SdkClientGetStoreRequest struct {
 	ctx    _context.Context
-	Client OpenFgaClient
+	Client *OpenFgaClient
 
 	options *ClientGetStoreOptions
 }
 
+type SdkClientGetStoreRequestInterface interface{
+	Options(options ClientGetStoreOptions) SdkClientGetStoreRequestInterface
+	Execute() (*ClientGetStoreResponse, error)
+
+    GetContext() _context.Context
+    GetOptions() *ClientGetStoreOptions
+}
+
 type ClientGetStoreOptions struct {
 }
 
 type ClientGetStoreResponse = openfga.GetStoreResponse
 
-func (request SdkClientGetStoreRequest) Options(options ClientGetStoreOptions) SdkClientGetStoreRequest {
+func (request *SdkClientGetStoreRequest) Options(options ClientGetStoreOptions) SdkClientGetStoreRequestInterface {
 	request.options = &options
 	return request
 }
 
-func (request SdkClientGetStoreRequest) Execute() (*ClientGetStoreResponse, error) {
+func (request *SdkClientGetStoreRequest) Execute() (*ClientGetStoreResponse, error) {
 	return request.Client.GetStoreExecute(request)
 }
 
-func (client *OpenFgaClient) GetStoreExecute(request SdkClientGetStoreRequest) (*ClientGetStoreResponse, error) {
-	data, _, err := client.OpenFgaApi.GetStore(request.ctx).Execute()
+func (request *SdkClientGetStoreRequest) GetContext() _context.Context{
+	return request.ctx
+}
+
+func (request *SdkClientGetStoreRequest) GetOptions() *ClientGetStoreOptions{
+	return request.options
+}
+
+func (client *OpenFgaClient) GetStoreExecute(request SdkClientGetStoreRequestInterface) (*ClientGetStoreResponse, error) {
+	data, _, err := client.OpenFgaApi.GetStore(request.GetContext()).Execute()
 	if err != nil {
 		return nil, err
 	}
 	return &data, nil
 }
 
-func (client *OpenFgaClient) GetStore(ctx _context.Context) SdkClientGetStoreRequest {
-	return SdkClientGetStoreRequest{
-		Client: *client,
+func (client *OpenFgaClient) GetStore(ctx _context.Context) SdkClientGetStoreRequestInterface {
+	return &SdkClientGetStoreRequest{
+		Client: client,
 		ctx:    ctx,
 	}
 }
 
 // / DeleteStore
 type SdkClientDeleteStoreRequest struct {
 	ctx    _context.Context
-	Client OpenFgaClient
+	Client *OpenFgaClient
 
 	options *ClientDeleteStoreOptions
 }
+
+type SdkClientDeleteStoreRequestInterface interface{
+	Options(options ClientDeleteStoreOptions) SdkClientDeleteStoreRequestInterface
+	Execute() (*ClientDeleteStoreResponse, error)	
+
```

Comment:
Something off with the formatting here, can you run gofmt?

Reply:
Same with several sections below

Reply:
Did not bring in https://github.com/openfga/sdk-generator/pull/131